scripts/r.fillnulls/r.fillnulls.py

```diff
@@ -473,7 +473,7 @@ def main():
                     flags='n',
                     stderr=subprocess.PIPE,
                     env=new_env)
-                stdout, stderr = p.communicate()
+                stderr = p.communicate()[1].decode()
```

Comment:
Please use decode function from library (grass.script.decode), it will use the local encoding instead of always utf-8.